pkgs/development/compilers/julia/1.5.nix

```diff
@@ -0,0 +1,163 @@
+{ stdenv, fetchurl, fetchzip, fetchFromGitHub
+# build tools
+, gfortran, m4, makeWrapper, patchelf, perl, which, python2
+, cmake
+# libjulia dependencies
+, libunwind, readline, utf8proc, zlib
+# standard library dependencies
+, curl, fftwSinglePrec, fftw, gmp, libgit2, mpfr, openlibm, openspecfun, pcre2
+# linear algebra
+, blas, lapack, arpack
+# Darwin frameworks
+, CoreServices, ApplicationServices
+}:
+
+assert (!blas.isILP64) && (!lapack.isILP64);
+
+with stdenv.lib;
+
+let
+  majorVersion = "1";
+  minorVersion = "5";
+  maintenanceVersion = "2";
```

Comment:
Is this really needed? Seems bloated to me.

Reply:
Not if you consider the passthru.